internal/controller/volsync/vshandler.go

```diff
@@ -60,6 +60,9 @@ const (
 
 	OwnerNameAnnotation      = "ramendr.openshift.io/owner-name"
 	OwnerNamespaceAnnotation = "ramendr.openshift.io/owner-namespace"
+
+	// StorageClass label
+	StorageIDLabel = "ramendr.openshift.io/storageid"
```

Comment:
Duplicate `StorageIDLabel`. One here and the other one in `volureplicationgroup_controller.go`.

Reply:
Yes